pkg/testframework/config.go

```diff
@@ -39,3 +46,89 @@ func GetConfig() (*rest.Config, error) {
 
 	return nil, fmt.Errorf("could not locate a kubeconfig")
 }
+
+// Reset the original cm stored in the openshift-config-managed namespace, if it exist
+func PutBackManagedConfigMap(t *testing.T, client *Clientset, ns string, cm *corev1.ConfigMap) {
```

Comment:
I'm not sure if you really need this wrapper function, your test function should know if the configmap existed from the first `GET` via `err != nil`, so using `if cm.Name != "" ` as a proxy is less than ideal:

```golang
func TestBrandingFromManagedConfigMap() {
  // I know I got a configmap, or I didn't. best to use the error as a signal and skip the func?
  originalManagedConfigMap, err := client.ConfigMaps(consoleapi.OpenshiftConfigManagedNamespace).Get(consoleapi.OpenShiftConsoleConfigMapName, metav1.GetOptions{})
   if err != nil {
      // no configmap originally
   }
   // make a copy so you can put it back
   configYouWillModify := originalManagedConfigMap.DeepCopy()
   // use update to put it back unmodified orig when done
   defer client.ConfigMaps(consoleapi.OpenshiftConfigManagedNamespace).Update(consoleapi.OpenShiftConsoleConfigMapName, metav1.GetOptions{})
   //
   // do your work...
   // do your work...
   //
}
```  
I think this probably reduces clarity.